packages/millicast-sdk/src/utils/Diagnostics.js

```diff
@@ -39,9 +40,14 @@ const getDiagnostics = (statsCount = MAX_STATS_HISTORY_SIZE) => {
   diagnostics.version = version
```

Comment:
I am thinking we should not return the global diagnostics object, instead should return a new object in this function when user invoke it. Otherwise, if user wants to keep two diagnostics records, the second function call will modify the first one.